### PR TITLE
Docs documenter

### PR DESCRIPTION
**Pull request by blas-ko**
Documented version of `TaylorSeries` Some documentation for `TaylorN.jl` is missing, and some changes are to be made in `Taylor1.jl`, but it's practically ready.

### CODE REVIEW

**Comment by blas-ko** [comment]
Changes were made in `Taylor1.jl`'s documentation so syntax is coherent in every function. I tried everything in a **Jupyter notebook** but not in the documentation page (the `make.jl` one).  On the notebook everything is working fine except for `? *(a,b)` which doesn't display anything.

**Comment by lbenet** [comment]
I'll check this later. Thanks any way!

File: src/Taylor1.jl [hunk 1]
```diff
@@ -225,6 +246,14 @@ end
 ## Division ##
 /{T<:Real}(a::Taylor1, b::T) = a * inv(b)
 /{T<:Complex}(a::Taylor1, b::T) = a * inv(b)
+doc"""
+    /(a,b)
+
+Return the Taylor expansion of $a/b$, of order `maximum(a.order,b.order)`, for
+`a::Taylor1`, `b::Taylor1` polynomials.
+
+For details on making the Taylor expansion, see [`TaylorSeries.mulHomogCoef`](@ref).
```
Comment (lbenet) [inline src/Taylor1.jl]: This should point to `TaylorSeries.divHomogCoef` !

File: src/Taylor1.jl [hunk 2]
```diff
@@ -661,79 +883,20 @@ end
 
 ## Returns de n-th derivative of a series expansion
```
Comment (lbenet) [inline src/Taylor1.jl]: This block (lines 884-894) is repeated and corresponds to the case `derivative(n:Int, a::Taylor1)`; please delete.

**Comment by lbenet** [comment]
I've sent you two comments on the code; once they are incorporated, I will merge. 

There are still things to be done, but the changes (e.g., diffTaylor -> derivative) are important to be in master.

**Comment by lbenet** [comment]
I'm closing this PR, because there is a mess of repeated commits. But I'm keeping your last additions in the "docs_documenter" branch. There are a couple of things to do to fully integrate Documenter.jl; once I'm done with that, I'll merge this.

**Pull request closed without merging**